src/common/distributionData.ts

```diff
@@ -0,0 +1,218 @@
+// symbol
+
+import { Address } from 'viem';
+
+export enum SUPPORTED_CHAINS {
+  BASE = 8453,
+}
+
+// NOTE: this data could be fetched from backend to avoid publish a new version with each update
+// this list coulkd change a lot so could be easiest in that way
+export const USERS_REWARDS: Record<string, Record<number, { address: string; amount: string }[]>> = {
```

Comment:
Please remove this constant from the sdk, as its for testing, it should be used inside a test case. The list of user rewards should be passed as a param to all the functions and should not be a part of sdk.